Data/TOR/Story/XML/10540.xml

```diff
@@ -599,63 +599,63 @@ anything of interest on my end.</EnglishText>
       <PointerOffset>2226</PointerOffset>
       <JapaneseText>ワン・ギン様のところの
 お迎えをやってるって？</JapaneseText>
-      <EnglishText/>
+      <EnglishText>You were going to pick it up at Wang Gin's place?</EnglishText>
```

Comment:
This is referring to when the cast has to pick up the guests for Wang Gin's auction. 
Maybe something like "You're escorting Wang Gin's guests?"